cardinal/persona/query/query_persona_signer.go

```diff
@@ -37,7 +38,7 @@ func PersonaSignerQuery(wCtx engine.Context, req *PersonaSignerQueryRequest) (*P
 	addr, err := wCtx.GetSignerForPersonaTag(req.PersonaTag, req.Tick)
 	if err != nil {
 		//nolint:gocritic // cant switch case this.
-		if errors.Is(err, ErrPersonaTagHasNoSigner) {
+		if strings.Contains(err.Error(), ErrPersonaTagHasNoSigner.Error()) {
```

Comment:
made this change just to get e2e to pass. I'd recommend updating the code to not have duplicate `ErrPersonaTagHasNoSigner` errors. We should define one, and import it where needed.